documents/payment.md

```diff
@@ -465,6 +465,101 @@ body, err := client.Payment.CreatePaymentJson(para_attr, nil)
   ]
 }
 ```
+-------------------------------------------------------------------------------------------------------
+### OTP Generate
+
+```go
+body, err := client.Payment.OtpGenerate(paymentId, nil, nil)
+```
+
+**Parameters:**
+
+| Name        | Type    | Description                          |
+|-------------|---------|--------------------------------------|
+| paymentId*    | integer | Unique identifier of the payment                                               |
+
+Doc reference [doc](https://razorpay.com/docs/payments/payment-gateway/s2s-integration/json/v2/build-integration/cards/#otp-generation-)
```

Comment:
Please remove the trailing hyphen in the url, it prevents jumping into the OTP section within the page.

Reply:
hyphen has been removed